### PR TITLE
feat: extracting indexes supports merge frame

### PR DESCRIPTION
**Pull request by vagetablechicken**
Support kFrameRowsMergeRowsRange, set it as absandlat.

### CODE REVIEW

File: src/base/ddl_parser_test.cc [hunk 1]
```diff
@@ -252,6 +282,25 @@ TEST_F(DDLParserTest, renameColumns) {
...
+        "      sum(col1) OVER (PARTITION BY pk1 ORDER BY std_ts ROWS_RANGE BETWEEN 30s PRECEDING AND CURRENT ROW) as "
+        "w3_col1_sum\n"
+        "      FROM t1\n"
+        "      WINDOW w2 AS (PARTITION BY pk1 ORDER BY std_ts ROWS BETWEEN 2 PRECEDING AND CURRENT ROW);";
+    auto index_map = DDLParser::ExtractIndexes(sql, db);
+    LOG(INFO) << index_map;
+    ASSERT_EQ(index_map.size(), 1);
+    auto index = index_map.begin()->second;
+    ASSERT_EQ(index.size(), 1);
+    ASSERT_EQ(index.begin()->ttl().ttl_type(), type::TTLType::kAbsAndLat);
```
Comment (dl239) [inline src/base/ddl_parser_test.cc]: check the value of  `abs_ttl` and `lat_ttl`

File: src/base/ddl_parser.cc [hunk 2]
```diff
@@ -304,29 +308,40 @@ bool IndexMapBuilder::UpdateIndex(const hybridse::vm::Range& range) {
...
+            DCHECK(frame->frame_rows() == nullptr && frame->frame_range());
+            // GetHistoryRangeStart is negative, ttl needs uint64
+            ttl_st_ptr->set_abs_ttl(ConvertToMinute(start));
+            ttl_st_ptr->set_ttl_type(type::TTLType::kAbsoluteTime);
+            break;
+        case hybridse::node::kFrameRowsMergeRowsRange:
+            DCHECK(frame->frame_rows() && frame->frame_range());
+            // use abs and ttl, only >abs_ttl and > lat_ttl will be expired
+            ttl_st_ptr->set_lat_ttl(rows_start);
+            ttl_st_ptr->set_abs_ttl(ConvertToMinute(start));
```
Comment (dl239) [inline src/base/ddl_parser.cc:340]: so the `abs_ttl` of `column_key` is minute?

Comment (vagetablechicken) [reply src/base/ddl_parser.cc:340]: After #597, it already does.

Comment (dl239) [reply src/base/ddl_parser.cc:340]: https://github.com/4paradigm/OpenMLDB/blob/main/src/cmd/sql_cmd.h#L1020 delete convert in HandleDeploy

Comment (vagetablechicken) [reply src/base/ddl_parser.cc:340]: ok

**Review by dl239 — APPROVED** [review]

**Review by zekai427 — APPROVED** [review]

**Pull request merged**